Assets/Scripts/Lua/CLRBindings/LuaButtons.cs

```diff
@@ -0,0 +1,154 @@
+﻿using UnityEngine;
+
+public class LuaButtons
```

Comment:
Is it possible for me to ask you to add the buttons to the newly created UI object instead? That way it's all in the same object, and buttons *are* part of the UI, after all.

Reply:
👌 ok